codegen/projections/rails_json/lib/rails_json/telemetry.rb

```diff
@@ -0,0 +1,925 @@
+# frozen_string_literal: true
+
+# WARNING ABOUT GENERATED CODE
+#
+# This file was code generated using smithy-ruby.
+# https://github.com/smithy-lang/smithy-ruby
+#
+# WARNING ABOUT GENERATED CODE
+
+module RailsJson
+  # @api private
+  module Telemetry
+
+    class AllQueryStringTypes
+      def self.in_span(context, &block)
+        attributes = {
+          'rpc.service' => 'RailsJson',
+          'rpc.method' => 'AllQueryStringTypes',
+          'code.function' => 'all_query_string_types',
+          'code.namespace' => 'RailsJson::Telemetry'
+        }
+        context.tracer.in_span('RailsJson.AllQueryStringTypes', attributes: attributes, kind: Hearth::Telemetry::SpanKind::CLIENT) do
+          block.call
```

Comment:
I don't think these need to call a block, the method could just return ` context.tracer.in_span('RailsJson.AllQueryStringTypes', attributes: attributes, kind: Hearth::Telemetry::SpanKind::CLIENT)` couldn't it? I would also format in_span multiline if it's easier to do so.